reversion/models.py

```diff
@@ -22,13 +23,17 @@
                                  _get_content_type, _get_options)
 
 
+logger = logging.getLogger(__name__)
+
+
 def _safe_revert(versions):
     unreverted_versions = []
     for version in versions:
         try:
             with transaction.atomic(using=version.db):
                 version.revert()
         except (IntegrityError, ObjectDoesNotExist):
+            logger.exception('Could not revert to {version}')
```

Comment:
```suggestion
            logger.warning('Could not revert to {version}', exc_info=True)
```

Reply:
Exceptions are logged at `ERROR` level, which is pretty serious, given this can often recover by iteself.

This edit still logs the traceback, but uses the `WARNING` level, which is more appropriate.